esp8266-fastled-webserver/Fields.cpp

```diff
@@ -120,130 +159,232 @@ String getSaturationBpm() {
+  brightDepthBpm = value.toInt();
+  return value;
 }
 
 String getBrightDepthMin() {
   return String(brightDepthMin);
 }
 String setBrightDepthMin(String value) {
-  brightDepthMin = value.toInt(); return value;
+  brightDepthMin = value.toInt();
+  return value;
 }
 
 String getBrightDepthMax() {
   return String(brightDepthMax);
 }
 String setBrightDepthMax(String value) {
-  brightDepthMax = value.toInt(); return value;
+  brightDepthMax = value.toInt();
+  return value;
 }
 
 String getBrightThetaIncBpm() {
   return String(brightThetaIncBpm);
 }
 String setBrightThetaIncBpm(String value) {
-  brightThetaIncBpm = value.toInt(); return value;
+  brightThetaIncBpm = value.toInt();
+  return value;
 }
 
 String getBrightThetaIncMin() {
   return String(brightThetaIncMin);
 }
 String setBrightThetaIncMin(String value) {
-  brightThetaIncMin = value.toInt(); return value;
+  brightThetaIncMin = value.toInt();
+  return value;
 }
 
 String getBrightThetaIncMax() {
   return String(brightThetaIncMax);
 }
 String setBrightThetaIncMax(String value) {
-  brightThetaIncMax = value.toInt(); return value;
+  brightThetaIncMax = value.toInt();
+  return value;
 }
 
 String getMsMultiplierBpm() {
   return String(msMultiplierBpm);
 }
 String setMsMultiplierBpm(String value) {
-  msMultiplierBpm = value.toInt(); return value;
+  msMultiplierBpm = value.toInt();
+  return value;
 }
 
 String getMsMultiplierMin() {
   return String(msMultiplierMin);
 }
 String setMsMultiplierMin(String value) {
-  msMultiplierMin = value.toInt(); return value;
+  msMultiplierMin = value.toInt();
+  return value;
 }
 
 String getMsMultiplierMax() {
   return String(msMultiplierMax);
 }
 String setMsMultiplierMax(String value) {
-  msMultiplierMax = value.toInt(); return value;
+  msMultiplierMax = value.toInt();
+  return value;
 }
 
 String getHueIncBpm() {
   return String(hueIncBpm);
 }
 String setHueIncBpm(String value) {
-  hueIncBpm = value.toInt(); return value;
+  hueIncBpm = value.toInt();
+  return value;
 }
 
 String getHueIncMin() {
   return String(hueIncMin);
 }
 String setHueIncMin(String value) {
-  hueIncMin = value.toInt(); return value;
+  hueIncMin = value.toInt();
+  return value;
 }
 
 String getHueIncMax() {
   return String(hueIncMax);
 }
 String setHueIncMax(String value) {
-  hueIncMax = value.toInt(); return value;
+  hueIncMax = value.toInt();
+  return value;
 }
 
 String getSHueBpm() {
   return String(sHueBpm);
 }
 String setSHueBpm(String value) {
-  sHueBpm = value.toInt(); return value;
+  sHueBpm = value.toInt();
+  return value;
 }
 
 String getSHueMin() {
   return String(sHueMin);
 }
 String setSHueMin(String value) {
-  sHueMin = value.toInt(); return value;
+  sHueMin = value.toInt();
+  return value;
 }
 
 String getSHueMax() {
   return String(sHueMax);
 }
 String setSHueMax(String value) {
-  sHueMax = value.toInt(); return value;
+  sHueMax = value.toInt();
+  return value;
+}
+
+
+typedef String (*FieldSetter)(String);
```

Comment:
SIMPLE FILE REVIEW

These are the other structures that were previously defined in `field.h`.